setup.cfg

```diff
@@ -1,18 +1,23 @@
 [tool:pytest]
 pep8maxlinelength = 119
+addopts = --cov-report term-missing --cov=logtracer --cov-branch -vv --cov-report=xml tests/
```

Comment:
'dd' typo?

Reply:
I assume you mean the -vv? If you do then:
> py.test -vv will show all of assert comparisons instead of truncating (https://docs.pytest.org/en/latest/announce/release-2.3.0.html)

Reply:
adopts or addopts?

Reply:
ah - read it as `add opts`, I was thrown off by that too!